### PR TITLE
[FIX] contract_payment_mode: get bank account from partner

### PR DESCRIPTION
**Pull request by jabibi**
in recurring invoice

### CODE REVIEW

File: contract_payment_mode/models/contract.py [hunk 1]
```diff
@@ -26,7 +26,9 @@ def _prepare_invoice_data(self, contract):
             _prepare_invoice_data(contract)
         if contract.payment_mode_id:
             invoice_vals['payment_mode_id'] = contract.payment_mode_id.id
-            invoice_vals['partner_bank_id'] = \
-                contract.payment_mode_id.bank_id.id
+            invoice_vals['partner_bank_id'] = (
+                contract.partner_id.bank_ids and
```
Comment (pedrobaeza) [inline contract_payment_mode/models/contract.py]: Use `contract.partner_id.bank_ids[:1].id or contract.payment_mode_id.bank_id.id)` for shorten the expression (and don't make another PR - I have told you how to do it in the other PR).

**Comment by jabibi** [comment]
Ok, but the odoo code use the other way, I had never seen your choice

**Comment by pedrobaeza** [comment]
All Odoo code is old. The new API gives this possibility.

**Comment by jabibi** [comment]
Give me 5 minutes to test your way

**Comment by jabibi** [comment]
Done
It is ok?

**Comment by pedrobaeza** [comment]
Yes, thanks!

:+1: 

Now we need more reviewers for this being merged.

**Comment by jabibi** [comment]
Ok, I will stay tuned for updates. Thanks for your help

**Comment by dreispt** [comment]
LGMT :+1:

**Comment by jabibi** [comment]
Just to know, what is the procedure to accept a PR? I read [this](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#please-respect-a-few-basic-rules), but I don't know if I miss something. Maybe it is not the place to ask

**Comment by pedrobaeza** [comment]
Well, we have 2 positive reviews and more than 5 days from the proposal, so we can merge. The problem is that as the reviews were done before the 5 days, there's no a tracker that warns about the 5 days expiration, so you have to write here for someone to notices, as you have done :wink:

**Comment by jabibi** [comment]
Thanks for the explanation @pedrobaeza. This is my first PR in OCA
